lvsfunc/comparison.py

```diff
@@ -483,7 +483,9 @@ def stack_planes(clip: vs.VideoNode, /, stack_vertical: bool = False) -> vs.Vide
 def tvbd_diff(tv: vs.VideoNode, bd: vs.VideoNode,
               thr: float = 72,
               height: int = 288,
-              return_array: bool = False) -> vs.VideoNode:
+              return_array: bool = False,
+              write_log: bool = False,
```

Comment:
i feel like we should just pass a file handle (maybe `typing.Optional[typing.TextIO]`?), then the caller can prepend/append any extra data they want (like episode name) as well as use a specific path